meta-balena-common/recipes-core/os-config/os-config/os-config.json

```diff
@@ -26,20 +26,15 @@
 		}
 	],
 	"keys": [
-		"applicationName",
 		"applicationId",
 		"deviceType",
-		"userId",
-		"username",
 		"appUpdatePollInterval",
 		"listenPort",
 		"vpnPort",
 		"apiEndpoint",
 		"vpnEndpoint",
 		"registryEndpoint",
 		"deltaEndpoint",
-		"pubnubSubscribeKey",
-		"pubnubPublishKey",
 		"mixpanelToken",
```

Comment:
Both pubnubSubscribeKey and pubnubPublishKey are used when starting the resin-supervisor at:

https://github.com/balena-os/meta-balena/blob/0d0decf620edd9ccc9e44041c2a4fbba91a8bde5/meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/start-resin-supervisor#L56

it's the same case as above, the entries here mean that os-config will remove these entries when transitioning from managed to unmanaged.

@majorz could you please chip in here?

Reply:
That usage should be removed - we haven't used pubnub in a long time now

Reply:
same here. I'll update the PR and remove this OS use